src/applications/static-pages/facilities/vet-center/createExpandableOperatingStatus.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default async function createExpandableOperatingStatus() {
+  const statusWidgets = Array.from(
+    document.querySelectorAll(
+      'div[data-widget-type^=expandable-operating-status]',
+    ),
+  );
+
+  if (statusWidgets.length) {
+    const {
+      default: ExpandableOperatingStatus,
+    } = await import('../../shared/ExpandableOperatingStatus');
```

Comment:
We should probably add a `webpackChunkName` comment here to support WebPack code splitting. We're doing this in our other React widgets.